applications/GeoMechanicsApplication/custom_processes/apply_constant_phreatic_multi_line_pressure_process.cpp

```diff
@@ -249,7 +249,7 @@ int ApplyConstantPhreaticMultiLinePressureProcess::findIndex(const Node& rNode)
         }
     }
 
-    return number_of_coordinates - 1;
+    return number_of_coordinates - 2;
```

Comment:
Nice find that we need to return the index of the 'one-before-the-last' instead of for the last point!

Reply:
All credits to @avdg81 ! I've just made what is written if the issue description.